dotCMS/src/main/java/com/dotmarketing/startup/runalways/Task00001LoadSchema.java

```diff
@@ -97,6 +98,8 @@ public void executeUpgrade() throws DotDataException, DotRuntimeException {
 					+ processedStatementCount + " statements", e);
 		}
 
+		LicenseManager.getInstance().reloadInstance();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)